configure.ac

```diff
@@ -1457,6 +1457,16 @@ AC_ARG_ENABLE(hwdb, [AS_HELP_STRING([--disable-hwdb], [disable hardware database
 AM_CONDITIONAL(ENABLE_HWDB, [test x$enable_hwdb = xyes])
 
 # ------------------------------------------------------------------------------
+AC_ARG_ENABLE([gudev],
```

Comment:
`--disable-gudev` has to be passed in order to configure successfully. Otherwise
```
checking for GLIB... no
configure: error: Package requirements (glib-2.0 >= 2.22.0 gobject-2.0 >= 2.22.0 gio-2.0) were not met:

No package 'glib-2.0' found
No package 'gobject-2.0' found
No package 'gio-2.0' found
```
This doesn't look like an "auto"-parameter.